tailwind.config.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
       "transit-green": {
         DEFAULT: "#E0EEEF",
         dark: "#017E84",
+        light: "#E0EEEF",
```

Comment:
#E0EEEF this color is already defined in  `DEFAULT: "#E0EEEF"`. If you want to use it just write in className="..-transit-green"